src/components/SlateEditor/plugins/conceptList/ConceptSearchResult.tsx

```diff
@@ -0,0 +1,133 @@
+import styled from '@emotion/styled';
+import { ConceptNotion, Spinner } from '@ndla/ui';
+import { spacing } from '@ndla/core';
+import { ConceptNotionType } from '@ndla/ui/lib/Notion/ConceptNotion';
+import { isNumber } from 'lodash';
+import { useEffect, useState } from 'react';
+import { useTranslation } from 'react-i18next';
+import config from '../../../../config';
+import { Embed } from '../../../../interfaces';
+import { searchPublishedConcepts } from '../../../../modules/concept/conceptApi';
+import { parseEmbedTag } from '../../../../util/embedTagHelpers';
+
+interface Props {
+  tag?: string;
+  language: string;
+  showResultCount?: boolean;
+}
+
+interface ConceptQuery {
+  tags: string;
+  language: string;
+  'page-size': number;
+}
+
+const StyledContentWrapper = styled.div`
+  & > figure:first-of-type {
+    margin-top: ${spacing.medium};
+  }
+`;
+
+const getVisualElement = (embed: Embed) => {
+  if (embed.resource === 'h5p') {
+    return {
+      resource: embed.resource,
+      url: embed.url,
+      title: embed.title,
+    };
+  }
+  if (embed.resource === 'brightcove') {
+    return {
+      resource: embed.resource,
+      url: `https://players.brightcove.net/${config.brightCoveAccountId}/${config.brightcovePlayerId}_default/index.html?videoId=${embed.videoid}`,
+      title: embed.title,
+    };
+  }
+  if (embed.resource === 'external') {
+    return {
+      resource: embed.resource,
+      url: embed.url,
+      title: embed.title,
+    };
+  }
+  if (embed.resource === 'image') {
+    return {
+      resource: embed.resource,
+      image: {
+        src: `${config.ndlaApiUrl}/image-api/raw/id/${embed.resource_id}`,
+        alt: embed.alt,
+      },
+    };
+  }
+};
+
+const ConceptSearchResult = ({ tag, language, showResultCount }: Props) => {
+  const [concepts, setConcepts] = useState<ConceptNotionType[]>([]);
+  const [loading, setLoading] = useState(false);
+  const [resultCount, setResultCount] = useState<number>();
+
+  const { t } = useTranslation();
+
+  const search = async (query: ConceptQuery) => {
+    searchPublishedConcepts(query)
```

Comment:
Trur ikkje det er så farlig å bruke publiserte forklaringer her, for vi viser upubliserte forklaringer ellers i artikkel-editor.

Reply:
Kan godt fjerne det igjen, men tenkte det mest riktige var å vise søket slik det faktisk kom til å se ut i artikkelen.

Reply:
Skjønner det, men vi sender en parameter til article-converter som forteller at vi skal bruke draft-forklaringer, for at forhåndsvisning skal være lik som i artikkelen. Då er det implisitt at publisert artikkel ikkje nødvendigvis viser alle forklaringene.